test/integration/guest_user_test.rb

```diff
@@ -3,8 +3,28 @@
 class GuestUserTest < ActionDispatch::IntegrationTest
   include Capybara::DSL
 
+  attr_reader :category, :item
+
+  def setup
+    @item = Item.create(name: 'espresso', cost: 9000)
+    @category = item.categories.create(name: 'hot beverages')
+  end
+
   test 'a guest user can view home page' do
     visit root_path
     assert page.has_content?('Coffee House')
   end
+
+  test 'a guest user can see all items' do
+    visit root_path
+    click_link_or_button('Menu')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.